README.md

```diff
@@ -299,6 +299,39 @@ const MyStyledDiv = glamorous.div(
 <MyStyledDiv /> // styles applied: {padding-top: 1, padding-right: 2, padding-bottom: 3, padding-left: 4} and anything coming from `extra-thing`.
 ```
 
+##### withProps
```

Comment:
I actually like this, as it opens the door for us to work with both:

```js
var Component = glamorous.div({backgroundColor: 'blue'})
var Component2 = Component.withProps({foo: 'bar'})
var Component3 = Component2.withStyle({color: 'red'})
```

Giving a consistent behavior to `with*` APIs